docs/en/Modules/Database-Tables.md

```diff
@@ -0,0 +1,1489 @@
+## AbpAuditLogs
```

Comment:
Add a heading and give a brief description for the purpose of this documentation. For example:

```suggestion
# Database Tables

This documentation describes all database tables and their purposes. You can read this documentation to get an overall knowledge of the database tables that comes from different modules.
```

> Please, use your own words I just added this as a suggestion.